doc/source/getting_started/licensing.rst

```diff
@@ -0,0 +1,83 @@
+.. _ref_licensing:
+
+=========
+Licensing
+=========
+
+This section details what the user should expect in terms of limitations or license usage
+when running PyDPF scripts.
+DPF follows a client-server architecture,
+which means that the PyDPF client library must interact with a DPF server.
+It thus either starts a DPF server via a local installation of DPF,
+or it connects to an already running local or remote DPF server.
+
+DPF is installed along with ANSYS since ANSYS 2021R1 (see :ref:`ref_compatibility`).
+It is also available as a standalone application (see :ref:`ref_getting_started_with_dpf_server`).
```

Comment:
This link should go to the `DPF Preview License Agreement`. Instead, it is going to the top of the linked page

Reply:
@rafacanton this link is supposed to send the user to the page dedicated to the DPF Server, not necessarily the License Agreement.